doc/lua_api.txt

```diff
@@ -4309,12 +4309,21 @@ Definition tables
 * `"image.png"`
 * `{name="image.png", animation={Tile Animation definition}}`
 * `{name="image.png", backface_culling=bool, tileable_vertical=bool,
-    tileable_horizontal=bool}`
+    tileable_horizontal=bool, align_style="node"/"world"/"user", scale=float}`
```

Comment:
`scale` is an integer now. Lua doesn't know that type but should corrected here for better understanding nonetheless.